[2022-06-10 04:42:17,156] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: hands_on_test.drop_order_table scheduled__2022-06-09T00:00:00+00:00 [queued]>
[2022-06-10 04:42:17,217] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: hands_on_test.drop_order_table scheduled__2022-06-09T00:00:00+00:00 [queued]>
[2022-06-10 04:42:17,219] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-06-10 04:42:17,220] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2022-06-10 04:42:17,222] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-06-10 04:42:17,310] {taskinstance.py:1377} INFO - Executing <Task(MySqlOperator): drop_order_table> on 2022-06-09 00:00:00+00:00
[2022-06-10 04:42:17,330] {standard_task_runner.py:52} INFO - Started process 189 to run task
[2022-06-10 04:42:17,344] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'hands_on_test', 'drop_order_table', 'scheduled__2022-06-09T00:00:00+00:00', '--job-id', '248', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpzbbj_ojr', '--error-file', '/tmp/tmprc37l8l0']
[2022-06-10 04:42:17,347] {standard_task_runner.py:80} INFO - Job 248: Subtask drop_order_table
[2022-06-10 04:42:17,497] {task_command.py:369} INFO - Running <TaskInstance: hands_on_test.drop_order_table scheduled__2022-06-09T00:00:00+00:00 [running]> on host b79bd65fcc57
[2022-06-10 04:42:17,940] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=hands_on_test
AIRFLOW_CTX_TASK_ID=drop_order_table
AIRFLOW_CTX_EXECUTION_DATE=2022-06-09T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-09T00:00:00+00:00
[2022-06-10 04:42:17,945] {mysql.py:82} INFO - Executing: 
DROP TABLE IF EXISTS orders;
[2022-06-10 04:42:17,978] {base.py:68} INFO - Using connection ID 'mysql_db' for task execution.
[2022-06-10 04:42:18,100] {dbapi.py:208} INFO - Running statement: 
DROP TABLE IF EXISTS orders;, parameters: None
[2022-06-10 04:42:18,104] {dbapi.py:216} INFO - Rows affected: 0
[2022-06-10 04:42:18,197] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=hands_on_test, task_id=drop_order_table, execution_date=20220609T000000, start_date=20220610T044217, end_date=20220610T044218
[2022-06-10 04:42:18,327] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-10 04:42:18,812] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-06-10 04:59:58,841] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: hands_on_test.drop_order_table scheduled__2022-06-09T00:00:00+00:00 [queued]>
[2022-06-10 04:59:58,862] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: hands_on_test.drop_order_table scheduled__2022-06-09T00:00:00+00:00 [queued]>
[2022-06-10 04:59:58,863] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-06-10 04:59:58,864] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2022-06-10 04:59:58,865] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-06-10 04:59:58,889] {taskinstance.py:1377} INFO - Executing <Task(MySqlOperator): drop_order_table> on 2022-06-09 00:00:00+00:00
[2022-06-10 04:59:58,895] {standard_task_runner.py:52} INFO - Started process 1274 to run task
[2022-06-10 04:59:58,905] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'hands_on_test', 'drop_order_table', 'scheduled__2022-06-09T00:00:00+00:00', '--job-id', '267', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpq6dq5p_r', '--error-file', '/tmp/tmp6k_tcn6d']
[2022-06-10 04:59:58,909] {standard_task_runner.py:80} INFO - Job 267: Subtask drop_order_table
[2022-06-10 04:59:59,127] {task_command.py:369} INFO - Running <TaskInstance: hands_on_test.drop_order_table scheduled__2022-06-09T00:00:00+00:00 [running]> on host b79bd65fcc57
[2022-06-10 04:59:59,382] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=hands_on_test
AIRFLOW_CTX_TASK_ID=drop_order_table
AIRFLOW_CTX_EXECUTION_DATE=2022-06-09T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-09T00:00:00+00:00
[2022-06-10 04:59:59,387] {mysql.py:82} INFO - Executing: 
DROP TABLE IF EXISTS orders;
[2022-06-10 04:59:59,432] {base.py:68} INFO - Using connection ID 'mysql_db' for task execution.
[2022-06-10 04:59:59,534] {dbapi.py:208} INFO - Running statement: 
DROP TABLE IF EXISTS orders;, parameters: None
[2022-06-10 04:59:59,555] {dbapi.py:216} INFO - Rows affected: 0
[2022-06-10 04:59:59,580] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=hands_on_test, task_id=drop_order_table, execution_date=20220609T000000, start_date=20220610T045958, end_date=20220610T045959
[2022-06-10 04:59:59,614] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-10 04:59:59,714] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-06-10 06:57:40,936] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: hands_on_test.drop_order_table scheduled__2022-06-09T00:00:00+00:00 [queued]>
[2022-06-10 06:57:40,957] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: hands_on_test.drop_order_table scheduled__2022-06-09T00:00:00+00:00 [queued]>
[2022-06-10 06:57:40,958] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-06-10 06:57:40,960] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2022-06-10 06:57:40,962] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-06-10 06:57:41,001] {taskinstance.py:1377} INFO - Executing <Task(MySqlOperator): drop_order_table> on 2022-06-09 00:00:00+00:00
[2022-06-10 06:57:41,009] {standard_task_runner.py:52} INFO - Started process 365 to run task
[2022-06-10 06:57:41,025] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'hands_on_test', 'drop_order_table', 'scheduled__2022-06-09T00:00:00+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpjrfa34ae', '--error-file', '/tmp/tmpwz2n2bqg']
[2022-06-10 06:57:41,028] {standard_task_runner.py:80} INFO - Job 5: Subtask drop_order_table
[2022-06-10 06:57:41,188] {task_command.py:369} INFO - Running <TaskInstance: hands_on_test.drop_order_table scheduled__2022-06-09T00:00:00+00:00 [running]> on host a0fd966c00bf
[2022-06-10 06:57:41,416] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=hands_on_test
AIRFLOW_CTX_TASK_ID=drop_order_table
AIRFLOW_CTX_EXECUTION_DATE=2022-06-09T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-09T00:00:00+00:00
[2022-06-10 06:57:41,422] {mysql.py:82} INFO - Executing: 
DROP TABLE IF EXISTS orders;
[2022-06-10 06:57:41,492] {base.py:68} INFO - Using connection ID 'mysql_db' for task execution.
[2022-06-10 06:57:41,791] {dbapi.py:208} INFO - Running statement: 
DROP TABLE IF EXISTS orders;, parameters: None
[2022-06-10 06:57:41,808] {dbapi.py:216} INFO - Rows affected: 0
[2022-06-10 06:57:41,953] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=hands_on_test, task_id=drop_order_table, execution_date=20220609T000000, start_date=20220610T065740, end_date=20220610T065741
[2022-06-10 06:57:42,063] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-10 06:57:42,401] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-06-10 07:17:24,872] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: hands_on_test.drop_order_table scheduled__2022-06-09T00:00:00+00:00 [queued]>
[2022-06-10 07:17:24,927] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: hands_on_test.drop_order_table scheduled__2022-06-09T00:00:00+00:00 [queued]>
[2022-06-10 07:17:24,932] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-06-10 07:17:24,933] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2022-06-10 07:17:24,935] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-06-10 07:17:24,960] {taskinstance.py:1377} INFO - Executing <Task(MySqlOperator): drop_order_table> on 2022-06-09 00:00:00+00:00
[2022-06-10 07:17:24,967] {standard_task_runner.py:52} INFO - Started process 1725 to run task
[2022-06-10 07:17:24,977] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'hands_on_test', 'drop_order_table', 'scheduled__2022-06-09T00:00:00+00:00', '--job-id', '51', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpqd8q2rb5', '--error-file', '/tmp/tmp37xabc69']
[2022-06-10 07:17:24,996] {standard_task_runner.py:80} INFO - Job 51: Subtask drop_order_table
[2022-06-10 07:17:25,095] {task_command.py:369} INFO - Running <TaskInstance: hands_on_test.drop_order_table scheduled__2022-06-09T00:00:00+00:00 [running]> on host a0fd966c00bf
[2022-06-10 07:17:25,355] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=hands_on_test
AIRFLOW_CTX_TASK_ID=drop_order_table
AIRFLOW_CTX_EXECUTION_DATE=2022-06-09T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-09T00:00:00+00:00
[2022-06-10 07:17:25,357] {mysql.py:82} INFO - Executing: 
DROP TABLE IF EXISTS orders;
[2022-06-10 07:17:25,379] {base.py:68} INFO - Using connection ID 'mysql_db' for task execution.
[2022-06-10 07:17:25,425] {dbapi.py:208} INFO - Running statement: 
DROP TABLE IF EXISTS orders;, parameters: None
[2022-06-10 07:17:25,436] {dbapi.py:216} INFO - Rows affected: 0
[2022-06-10 07:17:25,484] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=hands_on_test, task_id=drop_order_table, execution_date=20220609T000000, start_date=20220610T071724, end_date=20220610T071725
[2022-06-10 07:17:25,514] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-10 07:17:25,564] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
